### ISSUE TITLE
Pokemon Fun Fest Celebi request

### ISSUE THREAD

**Issue by risingPhil** [projectpokemon/EventsGallery#456]
Hi,

I'm the author of [PokeMe64](https://github.com/risingPhil/PokeMe64) and I was recently contacted by someone about the Pokémon Fun Fest Celebi. He asked me if my project's rendition of that pokémon was based on a legit save. (It's not, it's based on the [Bulbapedia entry](https://bulbapedia.bulbagarden.net/wiki/List_of_European_language_event_Pok%C3%A9mon_distributions_in_Generation_II#Pok%C3%A9mon_Fun_Fest_Celebi))

Turns out that this is the same guy who posted this topic: https://projectpokemon.org/home/forums/topic/49020-pok%C3%A9mon-fun-fest-celebi/

And now that I discovered this git repository, I'm wondering if someone here potentially has access to a legit save file with this Pokémon Fun Fest variant. After all, the Bulbapedia entry must have been based on one. The fact that it got described in detail on there makes me think it might still be out there somewhere.

I just wanted to check if there's a way I could help the guy. It appears like it means a great deal to him. If not, well, at least I tried :)

**Comment by kamronbatman** [comment]
I got one at the time, but my cart has been lost to time. It's the same as the one given out in Santa Monica at the mall during the same year.

**Comment by risingPhil** [comment]
yeah that's too bad. Seems like this one is very hard to come by.

**Comment by Donenzonen** [comment]
Might the save posted here help? The guy has some celebi's so i expect the event as well? https://projectpokemon.org/home/forums/topic/13226-pokemon-crystal-version-with-pcny-event-pokemon/

**Comment by kamronbatman** [comment]
I believe the event he is talking about uses Pokemon Vending Machine v1, while PCNY is Pokemon Vending Machine v2. It's possible that it used the same hardware, but based on what I personally remember and pictures of the redemption, I doubt it matches.

**Comment by Donenzonen** [comment]
I don't know, i'm from Europe and we had no chance on any Pokemon event at all...

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________
